src/components/Header/header.module.scss

```diff
@@ -0,0 +1,32 @@
+.container {
+  align-items: center;
+  background-color: #f0f0f0;
+  display: flex;
+  justify-content: space-between;
+  margin: 1rem;
+  padding: 1rem;
+
+  .restro__details {
+    display: flex;
+    flex-direction: column;
+    gap: 0.5rem;
+    .restro__title {
+      color: #000;

```

Comment:
define color in the constant.scss and use it here

Reply:
Addressed